src/HassClient/Client/Messages.cs

```diff
@@ -100,6 +100,14 @@ public class HassState
         [JsonPropertyName("last_changed")] public DateTime LastChanged { get; set; } = DateTime.MinValue;
         [JsonPropertyName("last_updated")] public DateTime LastUpdated { get; set; } = DateTime.MinValue;
         [JsonPropertyName("state")] public dynamic? State { get; set; } = "";
+        [JsonPropertyName("context")] public HassContext? Context { get; set; } = null;
+    }
+
+    public class HassContext
+    {
+        [JsonPropertyName("id")] public string Id { get; set; }

```

Comment:
Set default values please to remove warnings. We do warning free code as a code quality measure. Should be easy.  integer to 0, strings to empty strings and objects to nullable types that sets to null. 
Thank you for the contribution. Other than this looks good.